config/load.go

```diff
@@ -116,6 +120,38 @@ func (c *Config) checkConfig() error {
 	return nil
 }
 
+// Return Consul configuration from environment variables
+func (c *Config) setEnvConsulConfig() {
+	// Use the value of CONSUL_HTTP_ADDR if not empty
+	if envAddr := os.Getenv(api.HTTPAddrEnvName); envAddr != "" {
+		c.Consul.Address = envAddr
+
+		// Enable SSL if CONSUL_HTTP_ADDR contains 'https://'
+		if strings.Contains(envAddr, "https://") {
```

Comment:
it would be good to add a `ToLower`, in case somebody uses an uppercase https for some reason...

Reply:
Good catch!